src/test/groovy/org/gradle/wrapperupgrade/GradleWrapperUpgradePluginFuncTest.groovy

```diff
@@ -125,6 +125,51 @@ wrapperUpgrade {
         result.output.contains('Reusing configuration cache.')
     }
 
+    def "upgrade wrapper on wrapper-upgrade-gradle-plugin with dry run and optional Git arguments"() {
```

Comment:
Once `-s` does not happen by default anymore (see my other PR comment), we might not need the boolean called `wrapperUpgrade.unsignedCommits` anymore.